### PR TITLE
Cash 1099

### PR DESCRIPTION
**Pull request by BoulderBrains**
Fixing up the "Read full borrower details" link on desktop and mobile, fixing a few styles, passing through the loan.id for the redirect.

### CODE REVIEW

File: src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue [hunk 1]
```diff
@@ -110,8 +110,12 @@
 			<partner-info-panel v-if="hasPartner" :loan-id="loanId" />
 			<trustee-info-panel v-if="hasTrustee" :loan-id="loanId" />
 			<div>
-				<router-link to="">
-					Read full details
+				<hr>
+				<router-link
```
Comment (mcstover) [inline src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue]: Please add tracking to this link. "Lending", "Read full borrower details", loan.id or something like that...

Comment (mcstover) [reply src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue]: heres a better format:
`v-kv-track-event="['Lending', 'click-Read full borrower details', 'Profile Link', loanId, 'true']"`

File: src/components/LoanCards/BorrowerInfo/BorrowerInfoBody.vue [hunk 2]
```diff
@@ -55,7 +55,7 @@ export default {
 		},
 		readMoreLinkText: {
 			type: String,
-			default: 'Read more',
+			default: 'Read full borrower details',
```
Comment (emuvente) [inline src/components/LoanCards/BorrowerInfo/BorrowerInfoBody.vue]: If we don't want to change the default here you could also pass this as the `read-more-link-text` prop for this component in the desktop version of the expanded loan card

**Comment by BoulderBrains** [comment]
@mcstover Tracking now in place.
@emuvente Is this what you meant in your previous comment about passing this text via a prop?

File: src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue [hunk 3]
```diff
@@ -105,13 +105,27 @@
...
+					:to="`/lend/${loan.id}`"
+					class="featured-text full-details-link"
+					v-kv-track-event="[
+						'Lending',
+						'click-Read full borrower details',
+						'Profile Link',
+						loanId,
+						'true'
```
Comment (mcstover) [inline src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue]: We don't need to pass 'true' to activate both snowplow and ga in the Ui server.

File: src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue (hunk 1 above)
Comment (BoulderBrains) [reply src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue]: Fixed, true has been removed.

File: src/components/LoanCards/BorrowerInfo/BorrowerInfoBody.vue [hunk 4]
```diff
@@ -55,7 +55,7 @@ export default {
 		},
 		readMoreLinkText: {
 			type: String,
-			default: 'Read more',
+			default: 'Read full details',
```
Comment (ksekhri) [inline src/components/LoanCards/BorrowerInfo/BorrowerInfoBody.vue]: Can we revert the default?

File: src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue [hunk 5]
```diff
@@ -73,6 +73,7 @@
 						:max-use-length="200"
 						:use="loan.use"
 						:loan-id="loan.id"
+						read-more-text-link=""
```
Comment (ksekhri) [inline src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue]: `read-more-text-link="Read full details"`

**Pull request merged**